packages/opds-web-client/src/components/Book.tsx

```diff
@@ -237,6 +249,38 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
     return links;
   }
 
+  getMedium(book) {
+    if (!book.raw || !book.raw["$"]) {
+      return "";
+    }
+
+    return book.raw["$"]["schema:additionalType"].value;
```

Comment:
It's also possible `book.raw["$"]` exists but does not contain `"scheme:additionalType"`.